storybookjs/storybook
# Upgrade react-syntax-highlighter to pick up security patch upstream in highlight.js

Signed-off-by: Lucas Gonze <lucas@gonze.com>

Issue:

## What I did

Upgraded package to pick up a security patch. Note that I disclosed the details via Twitter DM. Also, this needs to be back-ported to the 5.3.x branch.

Ran `yarn run test`. Found no new test failures compared with the count in the `next` branch.

## How to test

- [ This has no new functionality that would require a test. ] Is this testable with Jest or Chromatic screenshots?
- [ No ] Does this need a new example in the kitchen sink apps?
- [ No ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.
